### DISCUSSION TITLE
Upgrading from @react-native-community/async-storage to @react-native-async-storage/async-storage

### DISCUSSION THREAD

**Discussion by arunnambissan** [react-native-async-storage/async-storage#1060] [Q&A]
My project has below version of the package, it uses old react native architecture and hermes disabled.

"@react-native-community/async-storage": "1.6.1",

I was trying to rename the package name and update it to latest version. I couldn't find a documentation for that, can anyone help?

**Comment by krizzu** [comment]
There are no upgrade steps required from your side, simply update the naming in package.json and install dependencies. Then you'll have to change the package name in code, where you import Async Storage

You can follow the instruction here:
https://react-native-async-storage.github.io/async-storage/docs/install

**Reply by arunnambissan** [reply]
Also I will need to change the package name in native side, right? 
Hope I can refer to this - https://github.com/react-native-async-storage/async-storage/commit/dfa0afa6c04d66ae704949c8beeb3d4a15d6d643#diff-000d4c7eb2d56846e1bf176a2be8823e6af03f9130053bfe3f5dd1210c9ca8aa

Will it be better to keep the `Linking.md` file in the repo, so that people using old arch can refer to it?

**Reply by krizzu** [reply]
Auto linking is available since 0.60, so I'd rather suggest updating RN at that point.
